app/code/Magento/InventoryCatalog/Test/Api/GetDefaultStock.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryCatalog\Test\Api;
+
+use Magento\InventoryApi\Api\Data\StockInterface;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+use Magento\Framework\Webapi\Rest\Request;
+
+/**
+ * Class GetDefaultStock
+ */
+class GetDefaultStock extends WebapiAbstract
```

Comment:
Pls, add Test suffix to file
And change dockblock like 'Test that default Stock is present after installation'